components/about.js

```diff
@@ -29,11 +31,52 @@ const Logo = styled.img`
 
 const CallToAction = LinkButton.extend`margin-top: 48px;`
 
-export default () =>
-  <Container>
-    <Logo src="/static/logo.svg" alt="ZDev logo" />
-    <Header>Website and App Developers</Header>
-    <CallToAction href="#contact" data-scroll="true">
-      Tell us about your project
-    </CallToAction>
-  </Container>
+const modalStyle = {
```

Comment:
Attempted this by replacing: 

`const modalStyle = {
  overlay: {
    position: 'fixed',
    top: 75,
    left: 0,
    right: 0,
    bottom: 0,
    backgroundColor: 'rgba(255, 255, 255, 0.75)',
  },
  content: {
    position: 'absolute',
    top: '40px',
    left: '40px',
    right: '40px',
    bottom: '40px',
    border: '1px solid #ccc',
    background: '#fff',
    overflow: 'auto',
    WebkitOverflowScrolling: 'touch',
    borderRadius: '4px',
    outline: 'none',
    padding: '20px',
  },
}
.........
<Modal isOpen={this.state.projectModalOpen} contentLabel="Project Query Modal" style={modalStyle}>
`
with:
`const ProjectQueryModal = styled(Modal)`
  overlay: {
    position: 'fixed',
    top: 75,
    left: 0,
    right: 0,
    bottom: 0,
    backgroundColor: 'rgba(255, 255, 255, 0.75)',
  },
  content: {
    position: 'absolute',
    top: '40px',
    left: '40px',
    right: '40px',
    bottom: '40px',
    border: '1px solid #ccc',
    background: '#fff',
    overflow: 'auto',
    WebkitOverflowScrolling: 'touch',
    borderRadius: '4px',
    outline: 'none',
    padding: '20px',
  }
`
...
<ProjectQueryModal isOpen={this.state.projectModalOpen} contentLabel="Project Query Modal">
`
But the styles weren't applied, I don't think the Modal component uses styles in a conventional way that would allow this to work.